dev/build-example-docs.py

```diff
@@ -64,6 +64,7 @@
 urls = {
     # Frameworks
     "Android": "https://www.android.com/",
+    "catboost": "https://catboost.ai/docs/en/",
```

Comment:
Is this key used somewhere? Do we need to update the casing somewhere else?

```suggestion
    "CatBoost": "https://catboost.ai/docs/en/",
```